examples/01-hello-world-cli/snapcraft.yaml

```diff
@@ -0,0 +1,16 @@
+name: hello-world-cli
+version: 0.1
+summary: Hello World simple C app
+description: |
+  This demo is intended to show how to build a C app
```

Comment:
Well, you can ignore the current examples directory, they are being renamed as "tour" and Mark is rewriting them.